cmake/copy_module_output.cmake

```diff
@@ -19,24 +19,26 @@ function(copy_module_output targetName targetDir)
 		make_directory ${RYTHE_DIR_OUTPUT_LIBS}/Release
 	)
 
-	# Copy debug library
-	# Note: || (exit 0) makes sure these commands succeed even if no library file was generated.
-	add_custom_command(TARGET ${targetName} 
-		POST_BUILD
-		COMMAND $<$<CONFIG:Debug>:${CMAKE_COMMAND}> -E 
-		copy 
-			$<TARGET_FILE:${targetName}>
-			"${RYTHE_DIR_OUTPUT_LIBS}/Debug/" || (exit 0)
-	)
+	if (${MODULE_INFO_HAS_SOURCES})
+		# Copy debug library
+		# Note: || (exit 0) makes sure these commands succeed even if no library file was generated.
```

Comment:
in which cases could that actually happen?

Reply:
Not many - it's possible for it to occur when the source itself has errors, in which case no .lib file gets generated, but the post build command could still try to run, which'll result in a difficult to understand error so I'd rather hide it I think